spec/site_github_munger_spec.rb

```diff
@@ -83,6 +84,10 @@
     before(:each) do
       Jekyll::GitHubMetadata.client = Jekyll::GitHubMetadata::Client.new({ :access_token => "" })
     end
+    before do
```

Comment:
Why not `before(:each)`? IIRC `before` is discouraged because of the random ordering of each test.